docs/running-on-mesos.md

```diff
@@ -387,6 +387,14 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.webui.url</code></td>
```

Comment:
I think I understand the intent, but the configuration setting here seems to be too general.
In the cases you changed the web ui, in the coarse/fine grain scheduler you're replacing the Spark App UI which is the default per Driver UI. In the Dispatcher case you're replacing the Dispatcher framework UI.
I think we want to be more specific here, one setting for the dispatcher uri another for the driver UI, which actually can be a generic setting not just for Mesos.